### ISSUE TITLE
Error in creating dir

### ISSUE THREAD

**Issue by H-Qiya** [gagneurlab/FRASER#20]
**Hi, Mr Mertes and your team!
I've read your manual and relative literature and found your package really worth a try of best effort!
However, when running your example data as well as my own data, I met the same error from the very first step as follows:**
**#######Example data in your manual#######**
> settings<-FraserDataSet(colData=sampleTable,workingDir=tempdir())
> settings
-------------------- Sample data table -----------------
## A tibble: 3 x 5
  sampleID bamFile                 group gene    pairedEnd
  <chr>    <chr>                   <int> <chr>   <lgl>    
1 sample1  extdata/bam/sample1.bam     1 TIMMDC1 TRUE     
2 sample2  extdata/bam/sample2.bam     3 CLPP    TRUE     
3 sample3  extdata/bam/sample3.bam     2 MCOLN1  TRUE     

----------------------- Settings -----------------------
Analysis name:               Data Analysis 
Analysis is strand specific: no 
Working directory:           'C:\Users\ADMINI~1\AppData\Local\Temp\RtmpQ9cxvH' 

-------------------- BAM parameters --------------------
Fri Dec 11 09:26:15 2020: The given working directory 'C:\Users\biocbuild\bbs-3.12-bioc\tmpdir\Rtmpw1fLb2/FRASER' does not exists. We will create it.
Error in h(simpleError(msg, call)) : 
  error in evaluating the argument 'object' in selecting a method for function 'scanBamParam': invalid class “FraserDataSet” object: Make sure we can write to the given working directory ' C:\Users\biocbuild\bbs-3.12-bioc\tmpdir\Rtmpw1fLb2/FRASER '.
In addition: Warning message:
In dir.create(object@workingDir, recursive = TRUE) :
  cannot create dir 'C:\Users\biocbuild', reason 'Permission denied'

**When working on my own data, I set my own working directory at the first step, and other script is almost the same. Nonetheless, the same error still exists:
#######My own data#######**
----------------------- Settings -----------------------
Analysis name:               Data Analysis 
Analysis is strand specific: no 
Working directory:           'Y:/HCM-newly-sequenced/200820-WGX' 
-------------------- BAM parameters --------------------
Fri Dec 11 09:51:35 2020: The given working directory 'C:\Users\biocbuild\bbs-3.12-bioc\tmpdir\Rtmpw1fLb2/FRASER' does not exists. We will create it.
Error in h(simpleError(msg, call)) : 
  error in evaluating the argument 'object' in selecting a method for function 'scanBamParam': invalid class “FraserDataSet” object: Make sure we can write to the given working directory ' C:\Users\biocbuild\bbs-3.12-bioc\tmpdir\Rtmpw1fLb2/FRASER '.
In addition: Warning message:
In dir.create(object@workingDir, recursive = TRUE) :
  cannot create dir 'C:\Users\biocbuild', reason 'Permission denied'

**Is the working directory completely fixed in ' C:\Users\biocbuild\bbs-3.12-bioc\tmpdir\Rtmpw1fLb2/FRASER ' ? How can I  alter it into my own path? Otherwise, must I run R as an administrator? Yet this might lead to unpredictable risk?
Looking forward to your reply!
Thanks a lot!**

**Comment by c-mertes** [comment]
Dear @H-Qiya, unfortunately, I can not reproduce your error. Even on Windows, I can not reproduce it. Which version do you use and how did you install it? Could you please try the code below in a fresh session?

You do not need to be an administrator and you set already correctly the working dir. So I'm very curious which version you use. 

```
BiocManager::install("FRASER")

library(FRASER)

sampleTable <- fread(system.file("extdata", "sampleTable.tsv", package = "FRASER", mustWork = TRUE))
bamFiles <- system.file(sampleTable[, bamFile], package = "FRASER", mustWork = TRUE)
sampleTable[, `:=`(bamFile, bamFiles)]

settings<-FraserDataSet(colData=sampleTable,workingDir=tempdir())
settings

BiocManager::valid()
```

This is run under R 4.0.2 and Bioconductor version 3.11 and FRASER 1.0.2

**Comment by H-Qiya** [comment]
Hi, @c-mertes !
Thanks a lot for your reply! What is unbelievable is that the Error in "---Bam parameters---" disappeared after copying your code above and running in R studio to reinstall FRASER! Meanwhile, my version is R 4.0.3, Bioconductor 3.12 and FRASER 1.2.0. And the code which reported error last time was copied from FRASER manual pdf.
Last time, I ran example data and my data respectively in R as an administrator to avoid the above "---Bam parameters---" error, though it was risky. And this time I ran the same data using FRASER without administrator mode. Then I compared those results and found they are exactly the same (I kept set.seed parameters consistent).
Anyway, this error has been fixed, though I didn’t figure out how it is  produced deeply in machine, hhh.

Also, I noticed your article has been published on NC the other day(DOI: 10.1038/s41467-020-20573-7). Congratulations on your achievement!!!
